yt_dlp/extractor/youtube.py

```diff
@@ -3913,11 +3913,28 @@ def _extract_from_tabs(self, item_id, ytcfg, data, tabs):
             playlist_id = channel_id
             tags = renderer.get('keywords', '').split()
 
-        thumbnails = (
+        primary_thumbnails = (
             self._extract_thumbnails(renderer, 'avatar')
             or self._extract_thumbnails(
                 primary_sidebar_renderer, ('thumbnailRenderer', 'playlistVideoThumbnailRenderer', 'thumbnail')))
 
+        channel_banners = self._extract_thumbnails(
+            data, ('header', ..., ['banner', 'mobileBanner', 'tvBanner']))
+        for banner in channel_banners:
+            banner['preference'] = -10
+
+        if channel_banners:
+            # We can get the uncropped banner by replacing the crop params with '=s0'
+            # See: https://github.com/yt-dlp/yt-dlp/issues/2237#issuecomment-1013694714
```

Comment:
that works for profile-pic too, according to the comment

Reply:
thanks, missed that. I might as well add that too